### ISSUE TITLE
RT5 compatibility?

### ISSUE THREAD

**Issue by athompso** [maettu/RT-Extension-BecomeUser#3]
Any chance of updating this for RT5?  The BPS-provided RT::Extension::SwitchUsers does not meet my needs, but it looks like this older module would.

**Comment by maettu** [comment]
@athompso I have not been using RT for years, but if you like to create a pull request or take over the module, that will be welcome.

**Comment by btb** [comment]
I made some changes for rt5, and it seems to be working fine:
https://github.com/btb/RT-Extension-BecomeUser/tree/rt5

I'd be happy to take over the module if you like.

**Comment by maettu** [comment]
@btb yes, absolutely. Let's make you co-maintainer on cpan, first. What is your cpan handle?

**Comment by btb** [comment]
Ah, I don't think I have one. I'll go sign up

**Comment by btb** [comment]
My CPAN handle has been set up now, it's "BTB"

**Comment by maettu** [comment]
Added you as co-maintainer. If there are any questions, please drop me a line.
